explorer/models/rootNode.ts

```diff
@@ -108,16 +105,15 @@ export class RootNode extends NodeBase {
 
     }
 
-    public async getChildren(element: NodeBase): Promise<NodeBase[]> {
-
+    public async getChildren(element: RootNode): Promise<NodeBase[]> {
```

Comment:
Usually use switch for equality alternatively use else if to avoid revaluation

Reply:
Changed to switch statements